Xplat/src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -132,9 +132,9 @@
   "botaniamisc.rannuncarpus.state_sensitive": "Match Exact State",
   "botaniamisc.rannuncarpus.state_insensitive": "Match Block Only",
   "botaniamisc.lokiRingLimitReached": "Selection limit reached",
-  "botaniamisc.loonium.generic_drops": "Not attuned",
-  "botaniamisc.loonium.structure_drops": "Attuned to structure",
-  "botaniamisc.loonium.multiple_structure_drops": "Attuned to %d structures",
+  "botaniamisc.loonium.generic_loot": "Generic loot",
+  "botaniamisc.loonium.structure_loot": "Structure loot",
+  "botaniamisc.loonium.custom_loot": "Customized loot",
```

Comment:
I think i don't actually like this. It's not really relevant to the player (in my eyes at least), whether the loot is from a structure or set via NBT. I think i liked "Attuned" better. Maybe it could be "Attuned to <structure>", (grabbing the structure name from the config (only if we make the config mandatory)) and you could choose your own string for that if you're setting the mobs with nbt.

Reply:
Yeah, which is why it would need to be specified in the config